core/nlp/spacy_tokenizer.py

```diff
@@ -14,19 +14,32 @@
 from spacy.language import Language
 from spacy.tokens import Doc, Token
 from spacy.vocab import Vocab
-from spacymoji import Emoji
+from spacy.lang.zh import Chinese
```

Comment:
Apologies, I had missed what you said here

> Made sure Chinese default tokenization is using jieba (Else it will only use character segmentation)

Let me resolve this.